doc/plugin_agent_workloadattestor_windows.md

```diff
@@ -0,0 +1,33 @@
+# Agent plugin: WorkloadAttestor "windows"
+
+The `windows` plugin generates Windows-based selectors for workloads calling the agent.
+It does so by opening an access token associated with the workload process. The system is then interrogated to retrieve user and group accounts information from that access token.
+
+### Workload Selectors
+
+| Selector | Value |
+| -------- | ----- |
+| `windows:user_sid` | The security identifier (SID) that identifies the user running the workload (e.g. `windows:user_sid:S-1-5-21-759542327-988462579-1707944338-1003`) |
+| `windows:user` | The user name of the user running the workload (e.g. `windows:user:computer-or-domain\myuser`) |
+| `windows:enabled_group_sid` | The security identifier (SID) that identifies an enabled group associated with the access token from the workload process (e.g. `windows:enabled_group_sid:S-1-5-21-759542327-988462579-1707944338-1004`) |
```

Comment:
I think that's important to be as explicit as possible about the properties/attributes from which the selectors are being generated. From that perspective, I think that having this selector as just `group_sid` would be very confusing and can lead to misinterpretations. I like the idea of changing it to be in the form `windows:group_sid:enabled:SOMEID`.

Reply:
Discussed some more with @MarcosDY about selector naming and meaning. Concluded that it would be good to include the value of the SE_GROUP_ENABLED attribute, which means that selectors will be generated for both true and false values. The shape would be:`windows:group_sid:se_group_enabled:true:SOME-SID`